module-cheatsheets/pathway-analysis-cheatsheet.md

```diff
@@ -0,0 +1,82 @@
+# Pathway Analysis Cheatsheet
+
+#### The tables below consist of valuable functions or commands that will help you through this module.
+##### Each table represents a different library/tool and its corresponding commands.
+> Please note that these tables are not intended to tell you all the information you need to know about each command.
+>
+> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command.
+
+<div style="page-break-after: always;"></div>
+
+### `msigdbr`
+
+Read the `msigdbr` documentation [**here**](https://www.rdocumentation.org/packages/msigdbr/versions/7.1.1).
+
+| Library/Package| Piece of Code| What it's called| What it does |
+|----------------|--------------|-----------------|--------------|
+| `msigdbr`| [`msigdbr()`](https://www.rdocumentation.org/packages/msigdbr/versions/7.1.1/topics/msigdbr)| geom vertical line| Retrieves the specified MSigDB dataset |
+
+### AnnotationDbi 
+Read the `AnnotationDbi` package vignette [**here**](http://bioconductor.org/packages/release/bioc/vignettes/AnnotationDbi/inst/doc/IntroToAnnotationPackages.pdf).
+
+| Library/Package                                  | Piece of Code                               | What it's called                               | What it does                                                                                                                                   |
+|-----------------------------------------|---------------------------------------------|-------------------------|------------------------------------------------------------------------|
+| `AnnotationDbi`        | [`keytypes()`](https://bioconductor.org/packages/release/bioc/vignettes/AnnotationDbi/inst/doc/IntroToAnnotationPackages.pdf)      | Keytypes     | Returns a character vector of column names/types of gene identifiers (e.g. `ENSEMBL` ) available in an `AnnotationDbi` package.                                              |
+| `AnnotationDbi`       | [`mapIDs()`](https://www.rdocumentation.org/packages/AnnotationDbi/versions/1.44.0/topics/AnnotationDb-objects) | Mapped IDs       | Extracts the mapped IDs for a set of gene identifiers. The types of gene identifiers for mapping are supplied to the arguments column (type of gene identifiers to return e.g. `ENSEMBL`) and keytype (type of gene identifiers to start with e.g. `ENTREZ`).                                  |  
```

Comment:
```suggestion
| `AnnotationDbi`       | [`mapIDs()`](https://www.rdocumentation.org/packages/AnnotationDbi/versions/1.44.0/topics/AnnotationDb-objects) | Mapped IDs       | Extracts the mapped IDs for a set of gene identifiers. The types of gene identifiers (e.g. `ENSEMBL` or `ENTREZ`) are supplied to arguments: `keytype` (type of gene identifiers we are providing in the `keys` argument) and  `column` (type of gene identifiers we want returned).                                  |  
```

I get confused by this everytime still; so I think this is good to explain :+1: I've just rearranged the wording a tad.

Reply:
`mapIDs()` -> `mapIds()`

Reply:
They used to have it the other way didn't they? I think that's why I'm getting confused.

Reply:
Sorry @cansavvy I only read this comment in email and did not notice that you caught the same thing!

Reply:
I didn't catch it. I edited the other part of the comment - missing that that was wrong. But then after you commented I  edited my suggestion.

Reply:
🤯